.env.ci

```diff
@@ -0,0 +1 @@
+BUILD_BACKENDS_PR_NUMBER="433"
```

Comment:
new line?

Reply:
This file is needed only to show that tests are compatible with that PR and will be removed before merge.